From c3436ec8a502a40579abf62be0a479fa9d7888ad Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Tue, 4 Jun 2019 11:39:39 +0800
Subject: [PATCH] multipath-tools: modify Makefile.inc for cross-compilation

Do not look for systemd info on the host, and allow us to pass in CFLAGS
using the OPTFLAGS variable.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Joe Slater <joe.slater@windriver.com>

Update for version 0.5.0-144-g770e6d0

Signed-off-by: Kai Kang <kai.kang@windriver.com>

Update for version 0.7.1

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

update for version 0.7.7

remove change about CFLAGS part, since patch 0024 already have similar function.

Signed-off-by: Changqing Li <changqing.li@windriver.com>

update for version 0.8.1
Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 Makefile.inc | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/Makefile.inc b/Makefile.inc
index 661f141..3f9fd28 100644
--- a/Makefile.inc
+++ b/Makefile.inc
@@ -35,17 +35,6 @@ ifndef RUN
 	endif
 endif
 
-ifndef SYSTEMD
-	ifeq ($(shell pkg-config --modversion libsystemd >/dev/null 2>&1 && echo 1), 1)
-		SYSTEMD = $(shell pkg-config --modversion libsystemd)
-	else
-		ifeq ($(shell systemctl --version >/dev/null 2>&1 && echo 1), 1)
-			SYSTEMD = $(shell systemctl --version 2> /dev/null | \
-				sed -n 's/systemd \([0-9]*\).*/\1/p')
-		endif
-	endif
-endif
-
 ifndef SYSTEMDPATH
 	SYSTEMDPATH=usr/lib
 endif
-- 
2.7.4

